app/assets/stylesheets/tinder.scss

```diff
@@ -0,0 +1,112 @@
+#tinderslide {
+  width: 1000px;
+  height: 550px;
+  margin: auto;
+  background-color: $background;
+  margin-top: 100px;
+
+  img {
+    float: left;
+    margin: 20px 20px 0 20px;
+    border-radius: 10px;
+
+    -webkit-box-shadow: 2px 2px 15px 2px #a2a2a2;
+    -moz-box-shadow: 2px 2px 15px 2px #a2a2a2;
+    box-shadow: 2px 2px 15px 2px #a2a2a2;
+  }
+
+  li {
+    box-shadow: 0 0 15px rgba(0, 0, 0, 0.04);
+    padding: 0 0 30px 0;
+    background: #FFF;
+    border-radius: 5px;
+    z-index: 0;
+  }
+}
+
+.actions a i {
+  font-size: 40px;
+}
+
+.actions a {
+  background-color: white;
+  background-image: none;
+  margin: 15px;
```

Comment:
Properties should be ordered background-color, background-image, border, border-radius, box-shadow, height, margin, width